command/agent_test.go

```diff
@@ -301,6 +305,7 @@ func TestAgent_HostBasedIDs(t *testing.T) {
 		cmd := &AgentCommand{
 			args: []string{
 				"-data-dir", tmpDir,
+				"-bind", "127.0.0.1",
```

Comment:
It tries to pick a private IP and fails because there is more than one. The exact error message is ```==> Failed to set up tagged and advertise addresses: Failed to get advertise address: Multiple private IPs found. Please configure one.
``` It doesn't look like a regression to me, maybe i am missing something here.  I get the same error starting consul on the master branch. 
What's changed is *where* it fails, with my changes it fails during readConfig, and that's why the test passes in the master branch.